src/transformers/configuration_distilbert.py

```diff
@@ -46,7 +90,6 @@ def __init__(
         attention_dropout=0.1,
         activation="gelu",
         initializer_range=0.02,
-        tie_weights_=True,
```

Comment:
@VictorSanh I did not see any usage of this argument so I removed it. Let me know if I missed something so that I may add it again if need be.

Reply:
hmm that's not normal. there should be a call to tie_weigths but it seems that it has been wrongly deleted. this is a bug.

Reply:
I didn't say anything. It's tied in `modeling_utils`. Yeah you can probably remove it.